src/cbuild/hooks/post_install/010_completions.py

```diff
@@ -0,0 +1,42 @@
+def invoke(pkg):
+    comps = []
+
+    if (
+        compspath := pkg.destdir / "usr/share/fish/vendor_completions.d"
+    ).is_dir():
+        comps.extend(
+            (f, f.name.removesuffix(".fish")) for f in compspath.iterdir()
+        )
+
+    if (compspath := pkg.destdir / "usr/share/zsh/site-functions").is_dir():
+        comps.extend(
+            (f, f.name[1:])
+            for f in compspath.iterdir()
+            if f.name.startswith("_")
+        )
+
+    if (
+        pkg.pkgname != "bash-completion"
+        and (
+            compspath := pkg.destdir / "usr/share/bash-completion/completions"
+        ).is_dir()
+    ):
+        comps.extend((f, f.name) for f in compspath.iterdir())
+
+    if (
+        compspath := pkg.destdir / "usr/share/nushell/vendor/autoload"
+    ).is_dir():
+        comps.extend(
+            (f, f.name.removesuffix(".nu")) for f in compspath.iterdir()
```

Comment:
technically this directory is not only for completions, but also for any kind of auto-loaded functions

this is also the case for zsh but in zsh's case it's mitigated by the convention of the underscore prefix for completion files i guess

Reply:
if we know it's broken and can be used for autoload then i think we should skip this dir entirely

Reply:
(for zsh, i think the underscore isn't only a convention but it also refuses to load another type of completion when pressing tab? i.e. if you have `foo` and name the completion anything other than `_foo`, it will not work, so it's ok)

Reply:
skipping nushell comps here sounds good to me, but this exact same issue crops up in the context of autosplitting `-nucomp` and `-zshcomp` packages; these packages currently take the functions dirs wholesale

Reply:
taking that is fine, the only issue is the name might be misleading. but it is indeed shell-specific data that is taken with the shell so the contents are correct and work fine

do you feel like renaming them to -nushelldata -zshdata or something? :)